packages/server/graphql/queries/helpers/fetchAtlassianProjects.ts

```diff
@@ -22,8 +21,8 @@ const fetchAtlassianProjects = async (
   }
 
   const cloudIds = sites.map((site) => site.id)
-  const atlassianProjects = [] as Omit<ISuggestedIntegrationJira, '__typename' | 'remoteProject'>[]
-  const service = 'jira' as any // TaskServiceEnum.jira
+  const atlassianProjects = [] as any
```

Comment:
`ISuggestedIntegrationJira` and `ISuggestedIntegrationGitHub` aren't being exported by Relay so I've added an `any` here